### DISCUSSION TITLE
I need help with compatibility with VR glasses!

### DISCUSSION THREAD

**Discussion by cld67** [Stellarium/stellarium#138] [Q&A] [answered]
I am new to this software and would really like to speak with someone regarding the capabilities, not only with VR glasses but use in a classroom setting.  
508-400-4349

**Comment by gzotti** [comment]
Developers with VR glasses are welcome to develop extensions and contribute them to the project.

**Comment by cld67** [comment]
I am having a hard time contacting the developers of the software I am interested in, (Stellarium has not gotten back to me and StarsightVR; is not available yet, although the developer did contact me)  I have been hired  to recommend VR headsets, with corresponding software that enables the astronomer to control what the "students" see.  I need non tethered headsets that are compatible with Stellarium

**Comment by cld67** [comment]
I am looking for guidance and opinions.  Please

**Comment by gzotti** [comment]
I had Oculus Rift and VIVE on my eyes for a few minutes each. For eyeglass wearers, the VIVE is better. However, I have no personal recent experience with developing with or for VR headsets. Our OR devkit was the wrong version to work with Starsight, so I did not follow that project further. IMHO StarsightVR is rather to be seen as forked 3rd party project (and whatever is done there is therefore "lost" to this project), not as part of Stellarium, and its developers should be contacted there. I am currently not involved with VR R&D but (like all Stellarium developers) busy with many other things, some of which feed me. I cannot support another weekend project, sorry. This may change with available funding, of course.

**Comment by alex-w** [comment, marked as answer]
Stellarium has Oculus plugin now

**Reply by alexistrigot** [reply]
Hi Alex, do you know where can I find this plugin ?
thanks !

**Reply by alex-w** [reply]
See https://github.com/Stellarium/stellarium/tree/master/plugins/Oculus

It's sad, but I have not touched this plugin yet due hardware limitation of PC for VR glasses support (Guillaume sent me VR glasses).
